index.html

```diff
@@ -226,6 +226,25 @@ <h2 class="primary-header">About me</h2>
       </ul>
 
     </section>
+    <section class="contact-form">
+      <h2 id="contact-header-message"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Let's replace the placeholder `{form-id}` with an actual id generated from formspree service. This way you will actually be able to recieve messages submitted in the contact form . Thanks :+1:

Reply:
Ok. Fixed.
Thanks you @sumairq .